packages/component/src/ActivityStatus/OthersActivityStatus.tsx

```diff
@@ -56,38 +55,14 @@ const OthersActivityStatus = memo(({ activity }: Props) => {
     }
   }, [graph, messageThing]);
 
-  const feedbackActions = useMemo<ReadonlySet<OrgSchemaAction> | undefined>(() => {
-    try {
-      const reactActions = (messageThing?.potentialAction || []).filter(
-        ({ '@type': type }) => type === 'LikeAction' || type === 'DislikeAction'
-      );
-
-      if (reactActions.length) {
-        return Object.freeze(new Set(reactActions));
-      }
-
-      const voteActions = graph.filter(({ type }) => type === 'https://schema.org/VoteAction').map(parseAction);
-
-      if (voteActions.length) {
-        return Object.freeze(new Set(voteActions));
-      }
-    } catch {
-      // Intentionally left blank.
-    }
-  }, [graph, messageThing]);
-
   return (
-    <Slotted className={classNames('webchat__activity-status', sendStatus + '')}>
-      {useMemo<ReactNode[]>(
-        () =>
-          [
-            timestamp && <Timestamp key="timestamp" timestamp={timestamp} />,
-            claimInterpreter && <Originator key="originator" project={claimInterpreter} />,
-            feedbackActions?.size && <Feedback actions={feedbackActions} key="feedback" />
-          ].filter(Boolean),
-        [claimInterpreter, timestamp, feedbackActions]
-      )}
-    </Slotted>
+    <span className={classNames('webchat__activity-status', className, sendStatus + '')}>
+      <StatusSlot>{timestamp && <Timestamp key="timestamp" timestamp={timestamp} />}</StatusSlot>
+      <StatusSlot>{claimInterpreter && <Originator key="originator" project={claimInterpreter} />}</StatusSlot>
+      <StatusSlot>
+        <ActivityFeedback activity={activity} key="feedback" placement="activity-status" />
```

Comment:
Instead of adding `placement` prop, should this line looks like:

```jsx
{
  feedbackActionsPlacement === 'activity-status' &&
    <StatusSlot>
      <ActivityFeedback activity={activity} key="feedback" />
    </StatusSlot>
}
```

Reply:
It needs to know the placement, so it can render larger icons.